docs/docs/authenticate/index.md

```diff
@@ -5,21 +5,61 @@ nav_order: 3
 has_children: true
 has_toc: false
 description: >
-  This page contains all the authentication methods that Firezone supports.
+  This page documents all the authentication methods that Firezone supports.
 ---
 ---
 
-Firezone provides the ability to require authentication to download device
-configuration files. Firezone supports the following single sign on (SSO)
-providers and authentication methods:
+Firezone can be configured to require authentication before users can generate
+or download device configuration files. Optionally,
+[periodic re-authentication](#enforce-periodic-re-authentication)
+can also be required for users to maintain their VPN session.
 
-* [Google]({%link docs/authenticate/google-sso.md%})
-* [Okta]({%link docs/authenticate/okta-sso.md%})
+![Azure SSO](https://user-images.githubusercontent.com/52545545/168922621-1f0f4dea-adfc-4e15-a140-a2f213676103.gif){:width="600"}
+
+By default, Firezone uses local email/password authentication, but can also
+support integration with any generic OpenID Connect
+(OIDC) identity provider. This allows users to sign in to Firezone using
+their credentials from Okta, Google, Azure AD, or your own custom identity provider.
+
+## Integrating A Generic OIDC Identity Provider
+
+The example below details the config settings required by Firezone to enable SSO
+through an OIDC identity provider. The configuration file can be found at
```

Comment:
nit: `OIDC identity provider` -> `OIDC provider`. I think this is the commonly used terminology around the web. `OIDC consumer` would be Firezone